docs/reference/management_commands.md

```diff
@@ -57,6 +57,17 @@ This command deletes old page revisions which are not in moderation, live, appro
 revision for a page. If the `days` argument is supplied, only revisions older than the specified number of
 days will be deleted.
 
+(purge_embeds)=
+
+## purge_embeds
+
+```sh
+manage.py purge_embeds
+```
+
+This command deletes all the cached embed objects from the database.
+It is required to run this command after everytime you make changes in embed settings for new changes to take effect.
```

Comment:
```suggestion
It is recimmended to run this command after changes are made to anyn embed settings so that subsequent embed usage does not from the database cache.
```